src/index.js

```diff
@@ -1 +1,49 @@
-module.exports = {};
+const hypernova = require('hypernova/server');
+const path = require('path');
+const winston = require('winston');
+
+const production = process.env.NODE_ENV === 'production';
+
+const defaultOptions = {
+  'assets-path': path.join(process.cwd(), 'dist'),
+  'logs-path': path.join(process.cwd(), 'dist', 'hypernova.log'),
+  'manifest-path': path.join(process.cwd(), 'dist', 'manifest.json'),
+  port: 13337,
```

Comment:
Before release, this should use the defaults from Hypernova -- less risk for breaking changes.